rubix-presto/pom.xml

```diff
@@ -32,6 +32,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs</artifactId>
+            <version>${dep.hadoop2.version}</version>
+            <scope>provided</scope>
```

Comment:
Why this is required ? There is no change in the presto module.

Reply:
My mistake. Thought this was required to resolve a dependency when trying to build after I merged. I'll remove this.